paasta_tools/cli/cmds/logs.py

```diff
@@ -167,7 +168,24 @@ def prefix(input_string, component):
     return "%s: %s" % (LOG_COMPONENTS[component]['color'](component), input_string)
 
 
-def paasta_log_line_passes_filter(line, levels, service, components, clusters):
+def check_timestamp_in_range(timestamp, start_time, end_time):
+    """A convenience function to check if a timestamp is within the given start and end times,
+    returns true if start_time or end_time is None
```

Comment:
On second thought, I prefer `None` instead of -Inf -> Inf since the API explicitly only handles either line based OR time based filtering. Mixing both of them is not allowed because it makes no sense, right now in the API if anywhere you encounter a start/end_time you can assume it will be None if we aren't filtering by it